tokio-quiche/src/http3/driver/streams.rs

```diff
@@ -106,6 +107,57 @@ impl StreamCtx {
             chan: self.recv.take(),
         })
     }
+
+    /// Handle the case when we received a STOP_SENDING frame. Note, that
```

Comment:
Can we leave a comment noting that Quiche will automatically respond to these with a RST_STREAM?